src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -63,9 +63,9 @@ public void storeMessage(Message message) {
    * @return a list of messages posted by the user, or empty list if user has never posted a
    *     message. List is sorted by time descending.
    */
-  public List<Message> getMessages(String recipient) {
+  public List<Message> getMessages(UUID postID) {
```

Comment:
does the project still compile with this line? other places in our code should be expected a String here instead of a UUID.